tests/yam/agama/validate_product_installed_agama.pm

```diff
@@ -0,0 +1,28 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Validate that the product installed by agama via /etc/os-release
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use Config::Tiny;
+use Test::Assert ':all';
+use scheduler 'get_test_suite_data';
+
+sub run {
+    select_console 'root-console';
+
+    my ($product) = @{get_test_suite_data()->{test_data}}{product};
+
+    my $os_release_output = script_output('cat /etc/os-release');
+    my $os_release_name = Config::Tiny->read_string($os_release_output)->{_}->{NAME};
```

Comment:
if you follow suggestion above, you might end up with two variables with `$os_release_name` so the way we usually disambiguate both is to use: `$os_release_name_current` and `$os_release_name_expected`. Also removing `_current` should be readable, because the enphasis in on having `_expected` in the name of variable, so it looks like proper testing :) (something that is missing in many places and in many validation in the repo, the future reader will thank us)